.jfrog-pipelines/release/pipelines.release.yml

```diff
@@ -57,15 +57,15 @@ pipelines:
             - git commit -am "[artifactory-release] Release version ${NEXT_VERSION} [skipRun]" --allow-empty
             - git tag ${NEXT_VERSION}
 
-            # Run build and publish
+            # Run validations, build and publish
             - >
-              jf gradle clean build artifactoryPublish -x test -x functionalTest
+              jf gradle clean validatePlugins build artifactoryPublish -x test -x functionalTest
```

Comment:
Can you add validatePlugins to the GitHub Actions tests?
We want to get feedback ASAP and not during release time.

Reply:
done, in addition added also for snapshot pipeline